src/table/create.rs

```diff
@@ -266,6 +267,49 @@ impl TableCreateStatement {
         self.indexes.as_ref()
     }
 
+    /// Rewriting extra param. You should take care self about concat extra params. Add extra after options.
+    /// Example for PostgresSQL [Citus](https://github.com/citusdata/citus) extension:
+    /// ```
+    /// use sea_query::{*, tests_cfg::*};
+    /// let table = Table::create()
+    ///     .table(Char::Table)
+    ///     .col(
+    ///         ColumnDef::new(Char::Id)
+    ///             .uuid()
+    ///             .extra("DEFAULT uuid_generate_v4()".into())
+    ///             .primary_key()
+    ///             .not_null(),
+    ///     )
+    ///     .col(
+    ///         ColumnDef::new(Char::CreatedAt)
+    ///             .timestamp_with_time_zone()
+    ///             .extra("DEFAULT NOW()".into())
+    ///             .not_null(),
+    ///     )
+    ///     .col(ColumnDef::new(Char::UserData).json_binary().not_null())
+    ///     .set_extra(&"USING columnar")
```

Comment:
I think this also work

```suggestion
    ///     .set_extra("USING columnar")
```